### DISCUSSION TITLE
v3.1.1

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#925] [Releases]
## Changes (v3.1.0)
* macOS Monterey support
* Fix warning message on OC boot (#585)
* SATA workaround is optional since it may break NVMe in some circumstances (#604)

## Changes (v3.1.1)
* Add Bluetooth support for Broadcom/Intel chips in Monterey (#923)

## Driver Updates (v3.1.0)
* AirportBrcmFixup 2.1.2 -> 2.1.3
* AppleALC 1.6.0 -> 1.6.4
* BrcmPatchRAM 2.5.8 -> 2.6.0
* IntelBluetoothFirmware 1.1.2 -> 2.0.0
* IntelMausi 1.0.6 -> 1.0.7
* Lilu 1.5.3 -> 1.5.6
* NVMeFix 1.0.7 -> 1.0.9
* OpenCore 0.6.9 -> 0.7.3
* OpenIntelWireless 1.3.0 -> 2.0.0
* Polaris22Fixup 1.3.3 -> 1.3.4
* VirtualSMC 1.2.3 -> 1.2.7
* WhateverGreen 1.4.9 -> 1.5.3

## Messages
* If you are upgrading from v2.17 and below, please read the notes at the top of [v3.0.1](https://github.com/osy/HaC-Mini/releases/tag/v3.0.1)
* Currently, secure boot [does not work on macOS 12](https://github.com/dortania/OpenCore-Legacy-Patcher/issues/471), if you wish to install or upgrade to macOS 12, you must disable secure boot first.
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model (a few stability updates per year). Following Apple's own deprecation model, we expect the EOL of this project to be no earlier than 2023.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.1.1'>v3.1.1</a>.</em>

**Comment by controlfix87** [comment]
I’ve upgraded my Wifi card to Intel AX210NWG. 
I’m running on BigSur 11.5.2 (latest) and it seems that the card is not getting a MAC address nor I can’t select any network (not showing any).
Installed the latest HacMini pkg 3.1.1 and selected intel wifi/bt, reset nvram twice. 
no luck on that. 
can you please help out to get my wifi working again? Before that I had an Intel 8265NGW and it worked great.

**Comment by rnlo** [comment]


**Reply by osy** [reply]
Are you trying to run RAID? That’s not supported.

**Reply by osy** [reply]
Did you reset your BIOS settings? It appears you’re running in RAID mode.

**Reply by osy** [reply]
Please open a new issue with the requested information on your configuration, etc. This is not the right place to report issues. 

https://github.com/osy/HaC-Mini/issues
